esp-hal/CHANGELOG.md

```diff
@@ -23,6 +23,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `Rng` and `Trng` now implement `Peripheral<P = Self>` (#2992)
 
 - `Async` drivers are no longer `Send` (#2980)
+- `flip-link` feature is now a config option
+- `flip-link` feature is now a config option (`ESP_HAL_CONFIG_FLIP_LINK`)
+- `flip-link` feature is now a config option (`ESP_HAL_CONFIG_FLIP_LINK`) (#3001)
+
+- Removed features `psram-quad` and `psram-octal` - replaced by `psram` and the `ESP_HAL_CONFIG_PSRAM_MODE` (`quad`/`octal`) (#3001)
```

Comment:
I think something has probably gone horribly wrong here 😁

Reply:
thanks - rebasing is making fun of me sometimes